src/cmakeProject.ts

```diff
@@ -2853,6 +2855,8 @@ export class CMakeProject {
             }
         }
 
+        await this.projectController?.updateActiveProject(this.workspaceFolder);
+
         // Regardless of the following configure return code,
         // we want full feature set view for the whole workspace.
         await enableFullFeatureSet(true);
```

Comment:
One thing I noticed while testing is that line 2863 (in the new version), will technically force a second configure if the `automaticReconfigure` setting is enabled. I don't think this needs to block this PR, but it is something we should look into.

Reply:
Will make a note to look into it in a follow up task